frontend/readhistory.lua

```diff
@@ -122,11 +122,11 @@ end
 
 --- Reads history table from file.
 -- @treturn boolean true if the history_file has been updated and reloaded.
-function ReadHistory:_read()
+function ReadHistory:_read(force_read)
     assert(self ~= nil)
     local history_file_modification_time = lfs.attributes(history_file, "modification")
     if history_file_modification_time == nil
-    or history_file_modification_time <= self.last_read_time then
+        or (not force_read and (history_file_modification_time <= self.last_read_time)) then
```

Comment:
The indentation was better before imo. Difference of opinion apparently. :-)

Reply:
For the first time I see you suggesting no indentation of the continuing line.

Reply:
(We've discussed this many times :) Personally, I don't like neither previous nor this one, and prefer the 3rd option below:
```lua
if some long stuff
or some other long stuff then
    blah = blah
end

if some long stuff
    or some other long stuff then
    blah = blah
end

if some long stuff
        or some other long stuff then
    blah = blah
end
```
)

Reply:
@hius07 
> For the first time I see you suggesting no indentation of the continuing line.

I didn't say I like the before, just that I like the after less. :-)

@poire-z left out some other possibilities, like for example:

```lua
if some long stuff
    or some other long stuff
then
    blah = blah
end
```

But I'm perfectly okay with his third.

Reply:
I kinda like aligning `or`/`and` with `if` in Lua ;).

But that changes between my mood. It's either that, or indenting the and/or *more*, like @Frenzie proposed (say, conditional +2).

Reply:
> I kinda like aligning or/and with if in Lua ;).

I don't dislike it either; I find it easy to read. It does change with the wind. :rofl: